integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/LogicalRelationVisitor.java

```diff
@@ -56,6 +62,7 @@ public boolean isDefinedAt(LogicalPlan x) {
     return x instanceof LogicalRelation
         && (((LogicalRelation) x).relation() instanceof HadoopFsRelation
             || ((LogicalRelation) x).relation() instanceof JDBCRelation
+            || ((LogicalRelation) x).relation() instanceof KafkaRelation
```

Comment:
Make a separate `KafkaRelationVisitor` class- otherwise, this code will throw a `NoClassDefFoundError` in any environment where the Kafka jar isn't present.

Reply:
I've add it. Then to have mutual exclusive visitors, we need to load classes by name:
https://github.com/OpenLineage/OpenLineage/pull/387/files#diff-c8683f5f8833efabf9d1d8a0accb97c3cc749ff4ddf64ca63b254d11d979b151R77
    https://github.com/OpenLineage/OpenLineage/pull/387/files#diff-56cd08a5821e71d48d554546eea9d746434ef50c4b53dae45064f152d4e7f27aR129